internal/resources/cloud/resource_cloud_stack.go

```diff
@@ -342,13 +342,7 @@ func readStack(ctx context.Context, d *schema.ResourceData, client *gcom.APIClie
 		return common.WarnMissing("stack", d)
 	}
 
-	connectionsReq := client.InstancesAPI.GetConnections(ctx, id.(string))
-	connections, _, err := connectionsReq.Execute()
-	if err != nil {
-		return apiError(err)
-	}
-
-	if err := flattenStack(d, stack, connections); err != nil {
+	if err := flattenStack(d, stack); err != nil {
```

Comment:
these changes were leftover from local dev and should be reverted from this PR

Reply:
Yes!